pkg/apis/metrics/v1alpha1/appmetricsapp_types.go

```diff
@@ -0,0 +1,42 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AppMetricsAppSpec defines the desired state of AppMetricsApp
+// +k8s:openapi-gen=true
+type AppMetricsAppSpec struct {
+	Name string `json:"appName"`
```

Comment:
It sounds like based on the conversation in the previous review, that this can be removed. Wdyt?

Reply:
Removed. 👍